### PR TITLE
Genoth rebase

### PR DESCRIPTION
**Pull request by gmassanek**
Rebased https://github.com/jlgoldman/membersofcongress/pull/8

### CODE REVIEW

File: members.json [hunk 1]
```diff
@@ -12482,10 +12612,10 @@
                 "line2": "6th Floor",
                 "longitude": -87.2143001,
                 "name": "Pensacola",
-                "phone": null,
+                "phone": "(850) 479-1183",
                 "state_code": "FL",
                 "type": "DISTRICT",
-                "zip": "32502"
+                "zip": "32503"
```
Comment (jlgoldman) [inline members.json]: Caught this in a spot check, the correct zip still seems to be 32502, do you know where this edit came from?

Comment (gmassanek) [inline members.json]: We'll check. I'm not sure offhand. @genoth do you know where this one came from?

**Comment by gmassanek** [comment]
I'll update my script to do a bit more verification before we merge. Should be able to get to it this week. Thanks!

File: members.json (hunk 1 above)
Comment (gmassanek) [inline members.json]: Fixed! I think this was just a typo

File: members.json [hunk 2]
```diff
@@ -551,11 +551,11 @@
...
                 "handle": "A000375-lubbock",
                 "latitude": 33.5885782,
-                "line1": "611 University Ave. #220",
-                "line2": null,
+                "line1": "1312 Texas Ave.",
+                "line2": "Suite 219",
                 "longitude": -101.8702325,
                 "name": "Lubbock",
-                "phone": null,
+                "phone": "(806) 763 - 1611",
```
Comment (jlgoldman) [inline members.json]: Sorry to nitpick, can you just normalize this phone number? All the rests looks good to go.

Thanks so much for following up!

Comment (gmassanek) [inline members.json]: Sure thing! This is fixed.

**Comment by gmassanek** [comment]
Once this merges, I'll rebase and get Sessions removed from https://github.com/jlgoldman/membersofcongress/pull/10

**Pull request merged**